Core/DISET/private/Transports/SSL/M2Utils.py

```diff
@@ -2,6 +2,9 @@
 """
 Utilities for using M2Crypto SSL with DIRAC.
 """
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
While these lines are almost surely innocuous, we don't want to repeat what was done with https://github.com/DIRACGrid/DIRAC/pull/4919 so unless they are needed, please do remove them.

IIRC the `print()` function that you are using below is also working in python 2.7.

Reply:
> IIRC the print() function that you are using below is also working in python 2.7.

Yup it'll be fine without the future import.

Reply:
OK, fine with me

Reply:
Yep but then py3 checks complain

```
************* Module DIRAC.Core.DISET.private.Transports.SSL.M2Utils
Core/DISET/private/Transports/SSL/M2Utils.py:60: [E1601(print-statement), ssl_verify_callback_print_error] print statement used
```

Reply:
I think we can reasonably disable that rule globally, we're already in the habit of not using print statements accidentally.

Reply:
Actually I misunderstood the check, yes we should just add the `__future__` imports in an identical way to v7r2 so they don't conflict when this is propagated.

Reply:
I'd rather be tempted to keep the warning (especially if we have new contributors (we can always dream (that starts looking like Scheme programming (https://en.wikipedia.org/wiki/Scheme_(programming_language))))) and re-add the import statements. That really shouldn't pose a merge conflict, if there is one, it should be straight forward to fix, and it is one of the extremely rare case where we actually need a print.